ecs-init/docker/docker_test.go

```diff
@@ -33,8 +33,8 @@ import (
 const (
 	testTempDirPrefix = "init-docker-test-"
 
-	expectedAgentBindsUnspecifiedPlatform = 21
-	expectedAgentBindsSuseUbuntuPlatform  = 18
```

Comment:
It not overly clear where these numbers come from, ideally these would come from the length of some slices or maps if they really matter